.github/workflows/Build-Rock.yaml

```diff
@@ -1,54 +1,65 @@
-name: Build rock
+name: Build Rock
 
 on:
   workflow_call:
     inputs:
+      # build parameters
       oci-archive-name:
-        description: "Final filename of the rock's OCI archive"
-        type: string
-        required: true
-      oci-factory-path:
-        description: "Path, in the OCI Factory, to this rock"
-        type: string
-        required: true
-      rock-name:
```

Comment:
I moved them to the Image workflow because it seemed to have better separation of responsibilities, especially if the build-rock workflow is used for external tasks. We can keep the Build-Rock workflow for just building the rock, and the Image workflow for the factory pipeline itself.